amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4307,26 +4321,100 @@ definitions:
     properties:
       name:
         type: string
+        description: |
+          The valid names are:
+          * retain
+          * cutoff
+          * accession
+          * transfer
+          * destroy
```

Comment:
```suggestion
          * destroyNode
          * destroyContent
```

Reply:
We have removed destroyNode and destroyContent and we kept one property named as retainRecordMetadataAfterDestruction. So now we should be good only one destroy step.